assets/src/edit-story/components/library/panes/media/common/innerElement.js

```diff
@@ -165,15 +139,14 @@ function InnerElement({
         alt={alt}
         aria-label={alt}
         loading={'lazy'}
-        onClick={onClick(thumbnailURL, mediaBaseColor.current)}
         onLoad={makeMediaVisible}
-        {...dropTargetsBindings(thumbnailURL)}
+        draggable={false}
       />
     );
+    cloneProps.src = thumbnailURL;
   } else if (type === 'video') {
-    const { lengthFormatted, poster, mimeType } = resource;
     const displayPoster = poster ? poster : newVideoPosterRef.current;
-    return (
+    media = (
       <>
         <Video
```

Comment:
I would expect this video element to stop playing, once I start dragging it (a clone of it, that is) in - that it keeps playing seem weird:

![video-drag-playing](https://user-images.githubusercontent.com/637548/101856682-cafed000-3b33-11eb-98ee-c8ecddb77a8d.gif)